client/keys.go

```diff
@@ -385,7 +385,9 @@ func (k *Key) sealDirect(sensitive []byte, opts sealOptsDirect) (*pb.SealedBytes
 	var pcrs *pb.PCRs
 	var err error
 	var auth []byte
-
+	if !k.pubAreaDirect.ObjectAttributes.Decrypt {
```

Comment:
You could also check that it's a restricted key, if you want.

Reply:
Check added, resolved.